test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -229,6 +304,32 @@ class test_ucp_sockaddr : public ucp_test {
         }
     }
 
+    void wait_for_server_reject(bool wakeup)
+    {
+        ucs_time_t time_limit = ucs_get_time() +
+                                ucs_time_from_sec(UCP_TEST_TIMEOUT_IN_SEC);
+
+        while ((receiver().get_rejected_cntr() == 0) &&
+               (ucs_get_time() < time_limit)) {
+            check_events(sender().worker(), receiver().worker(), wakeup, NULL);
+        }
+        EXPECT_GT(time_limit, ucs_get_time());
+        EXPECT_EQ(1ul, receiver().get_rejected_cntr());
+    }
+
+    void wait_for_client_reject(bool wakeup)
+    {
+        ucs_time_t time_limit = ucs_get_time() +
+                                ucs_time_from_sec(UCP_TEST_TIMEOUT_IN_SEC);
+
+        while ((sender().get_rejected_cntr() == 0) &&
+               (ucs_get_time() < time_limit)) {
+            check_events(sender().worker(), receiver().worker(), wakeup, NULL);
+        }
+        EXPECT_GT(time_limit, ucs_get_time());
+        EXPECT_EQ(1ul, sender().get_rejected_cntr());
```

Comment:
what should trigger the reject here?

Reply:
[Err callback](https://github.com/openucx/ucx/pull/2672/files#diff-c19a05052b898dabf25801d90564171bR418) with status `UCS_ERR_REJECTED` increments the counter